jax/_src/lax/control_flow/conditionals.py

```diff
@@ -87,6 +87,7 @@ def switch(index, branches, *operands):
   Args:
     index: Integer scalar type, indicating which branch function to apply.
     branches: Sequence of functions (A -> B) to be applied based on ``index``.
+      The output shapes of all branches must match.
```

Comment:
In general, the branches may return pytrees. Maybe we should say "All branches must return the same output structure"?

Reply:
Thanks! Done.